### PR TITLE
feat: remove old deployment scripts

### PR DESCRIPTION
**Pull request by deanamiel**
Remove old deployment scripts, link to contract deployments repository for up to date deployment scripts as per [AXE-3049](https://axelarnetwork.atlassian.net/jira/software/c/projects/AXE/boards/21?selectedIssue=AXE-3049)

[AXE-3049]: https://axelarnetwork.atlassian.net/browse/AXE-3049?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by deanamiel** [comment]
I removed all deployment scripts but are there any other scripts in this repository that are outdated and should be removed?

**Review by re1ro — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -93,39 +99,43 @@ npx hardhat test --network example --grep 'AxelarGateway'
...
-            getGasOptions(),
-        );
+const txExecute = await interchainGovernance.execute(commandIdGateway, governanceChain, governanceAddress, payload, getGasOptions());
 const receiptExecute = await txExecute.wait(network.config.confirmations);
- ```
+```
+
+-   The `changeEtherBalance` check expects one tx in a block so change in balances might need to be tested explicitly for unit tests using `changeEtherBalance`.
+
+## Deploying contracts
```
Comment (milapsheth) [inline README.md:132]: move this earlier, right after the build/test section

File: scripts/deploy-gateway-v3.2.x.js
Comment (milapsheth) [inline scripts/deploy-gateway-v3.2.x.js:1]: only flatten-contracts script is relevant. everything else we have scripts in the deployments repo to help with

File: README.md (hunk 1 above)
Comment (deanamiel) [reply README.md]: sounds good

File: scripts/deploy-gateway-v3.2.x.js
Comment (deanamiel) [reply scripts/deploy-gateway-v3.2.x.js:1]: removed all other scripts

**Review by milapsheth — APPROVED** [review]

**Pull request merged**